IntelFsp2WrapperPkg/Library/FspWrapperMultiPhaseProcessLib/PeiFspWrapperMultiPhaseProcessLib.c

```diff
@@ -288,6 +292,15 @@ FspWrapperVariableRequestHandler (
     }
   }
 
+  //
+  // Refresh FspHobList pointer stored in HOB.
+  //
+  GuidHob = GetFirstGuidHob (&gFspHobGuid);
+  if (GuidHob != NULL) {

```

Comment:
can we add an assertion ASSERT (GuidHob != NULL)?
If-check is ok to be kept.

Reply:
Done. Thanks for the comment.